### ISSUE TITLE
Unknown method invocation `toBytes` on String type

### ISSUE THREAD

**Issue by heuermh** [nf-core/viralrecon#142]
```
$ nextflow run main.nf -profile test,docker
...
[b2/a44cc7] process > output_documentation                                           [100%] 1 of 1 ✔
Unknown method invocation `toBytes` on String type -- Did you mean?
  getBytes

 -- Check script 'main.nf' at line: 3457 or see '.nextflow.log' file for more details
```

**Comment by drpatelh** [comment]
Thanks for posting this @heuermh Some more info would be useful though 😅 I am assuming you are running the `dev` version of the pipeline because that points to https://github.com/nf-core/viralrecon/blob/1779842a284772705ff0072b807d492f05b111e9/main.nf#L3457 as being the issue here. However, this is a little odd because the GitHub Actions CI tests are run in exactly the same way and passed on the last PR...

**Comment by heuermh** [comment]
Yep, am still in investigation mode, `dev` from git HEAD fails for me on OSX (#143) but not on linux (which succeeds but notes this issue).

**Comment by drpatelh** [comment]
Ok. This is quite scary because the CI tests passed but do indeed show the same error you just reported https://github.com/nf-core/viralrecon/runs/1453984332#step:7:215 . Its because I changed the type of `params.max_multiqc_email_size` in `nextflow.config` from a Nextflow memory unit to a string to deal with the new JSON schema 🤦

**Comment by drpatelh** [comment]
This is the fix by the way:
https://github.com/nf-core/rnaseq/blob/bc5fc76f40b2da6082a854927184c9d6e5060393/lib/Completion.groovy#L83

**Comment by heuermh** [comment]
Fixed by #144
